### PR TITLE
Add new 'See a GP' conditions

### PR DESCRIPTION
**Pull request by teneightfive**
Adding new conditions within the "Seeing a GP" cluster:

- [x] Broken or bruised ribs
- [x] Carpal tunnel syndrome
- [x] Cellulitis
- [x] Conjunctivitis
- [x] Frozen shoulder
- [x] Hiatus hernia
- [x] Impetigo
- [x] Insomnia
- [x] Iron deficiency anaemia
- [x] Slipped disk
- [x] Tendonitis
- [x] Urinary tract infections

### CODE REVIEW

**Comment by tyom** [comment]
Looks like there are problems with manifest files. We're both creating files with fragments and embedding them into manifest files. Also all URLs are `localhost`.

File: content/conditions/broken-or-bruised-ribs/manifest.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{
+  "layout": "content-simple",
+  "description": "Bruised or broken ribs can be very painful but usually heal by themselves.",
+  "choicesOrigin": "conditions/rib-injuries/pages/introduction.aspx",
```
Comment (tyom) [inline content/conditions/broken-or-bruised-ribs/manifest.json]: This should be an absolute URL, shouldn't it?

File: content/conditions/cellulitis/content-7.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Call 999 or go to A&E now if you have cellulitis with:
```
Comment (tyom) [inline content/conditions/cellulitis/content-7.md]: Use abbreviation?

File: content/conditions/conjunctivitis/content-3.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+## If you’re unsure it’s conjunctivitis
+
+Other conditions can cause [red eyes.](http://www.nhs.uk/conditions/red-eye/Pages/Introduction.aspx)
```
Comment (tyom) [inline content/conditions/conjunctivitis/content-3.md]: I think we agreed that full stops should be outside of links.

File: content/conditions/cellulitis/content-7.md (hunk 2 above)
Comment (teneightfive) [inline content/conditions/cellulitis/content-7.md]: Good spot. Will make sure it's added in all files that include that abbreviation. We should also find a better way to parse common abbreviations. Something more globally.

File: content/conditions/broken-or-bruised-ribs/manifest.json (hunk 1 above)
Comment (teneightfive) [inline content/conditions/broken-or-bruised-ribs/manifest.json]: No, `http://www.nhs.uk/` gets applied in the [new site message template](https://github.com/nhsuk/betahealth/blob/develop/app/views/_includes/new-site-message.nunjucks#L9).

File: content/conditions/conjunctivitis/content-3.md (hunk 3 above)
Comment (teneightfive) [inline content/conditions/conjunctivitis/content-3.md]: Yep, this was missed. Wonder if we can run something on CI to check?

**Pull request merged**